swarmrl/engine/espresso.py

```diff
@@ -214,6 +217,32 @@ def _check_already_initialised(self):
                 "after the first call to integrate()"
             )
 
+    def reset_system(self):
+        """
+        Reset the system to its initial state.
+        """
+        # save the old register
+        old_register = np.copy(self.colloid_radius_register).item()
+        # clear the system
+        self.colloid_radius_register = {}
+        self.system.part.clear()
+        self.colloids = list()
```

Comment:
Very pedantic comment so feel free to ignore, but you have:

```python
self.colloid_radius_register = {}
```
where you instantiate with the object `{}` which could be `self.colloid_radius_register = dict()` but then you have:
```python
self.colloids = list()
```
which could be `self.colloids = []`.

Only noticed because the two things are next to each other and it makes no difference it is just odd to use one paradigm and then another one after.